src/NodaTime.Test/Text/LocalDateTimePatternTest.cs

```diff
@@ -97,8 +97,10 @@ public class LocalDateTimePatternTest : PatternTestBase<LocalDateTime>
             new Data(MsdnStandardExample) { Pattern = "r", Text = "2009-06-15T13:45:30.090000000 (ISO)", Culture = Cultures.EnUs },
             new Data(SampleLocalDateTimeCoptic) { Pattern = "r", Text = "1976-06-19T21:13:34.123456789 (Coptic)", Culture = Cultures.EnUs },
             // Note: No RFC1123, as that requires a time zone.
-            // Sortable / ISO8601
+            // Short Sortable / ISO8601
             new Data(MsdnStandardExampleNoMillis) { Pattern = "s", Text = "2009-06-15T13:45:30", Culture = Cultures.EnUs },
+            // Long Sortable / ISO8601-extended
+            new Data(MsdnStandardExample) { Pattern = "S", Text = "2009-06-15T13:45:30.09", Culture = Cultures.EnUs },
```

Comment:
Perhaps add the NoMillis input here too to show that the decimals are truncated?

Reply:
Done.